src/components/ReserveSubheader.tsx

```diff
@@ -18,15 +19,21 @@ export function ReserveSubheader({ value, rightAlign }: ReserveSubheaderProps) {
         justifyContent: 'center',
       }}
     >
-      <FormattedNumber
-        compact
-        value={value}
-        variant="secondary12"
-        color="text.secondary"
-        symbolsVariant="secondary12"
-        symbolsColor="text.secondary"
-        symbol="USD"
-      />
+      {value === 'Disabled' ? (
+        <Typography component="span" sx={{ mr: 0.5 }} variant={'secondary12'} color={'text.muted'}>
```

Comment:
In all prop cases, they don't have to be wrapped in brackets if the value is just a string.  So `variant="secondary12"` and `color="text.muted"` suffice.  This is how we write it in the rest of the codebase as well.  TMYK